frontend/src/pages/modelServing/screens/metrics/performance/ModelMeshMetrics.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
+import MetricsChart from '~/pages/modelServing/screens/metrics/MetricsChart';
+import {
+  ModelMetricType,
+  ModelServingMetricsContext,
+} from '~/pages/modelServing/screens/metrics/ModelServingMetricsContext';
+import { ContextResourceData, PrometheusQueryRangeResultValue } from '~/types';
+import EnsureMetricsAvailable from '~/pages/modelServing/screens/metrics/EnsureMetricsAvailable';
+
+const ModelMeshMetrics: React.FC = () => {
+  const { data } = React.useContext(ModelServingMetricsContext);
+
+  return (
+    <Stack hasGutter>
+      <EnsureMetricsAvailable
+        metrics={[ModelMetricType.REQUEST_COUNT_SUCCESS, ModelMetricType.REQUEST_COUNT_FAILED]}
+        accessDomain="model metrics"
+      >
+        <StackItem>
+          <MetricsChart
+            metrics={[
+              {
+                name: 'Successful',
+                metric: data[
+                  ModelMetricType.REQUEST_COUNT_SUCCESS
+                ] as ContextResourceData<PrometheusQueryRangeResultValue>,
```

Comment:
Nit, not a blocker - I wonder if we can do something to avoid `as` at the usage level for this data. Maybe in `ModelServingMetricsContext` and `useModelServingMetrics` instead of using this:
```ts
type ModelServingMetricsContextType = {
  data: Record<
    ModelMetricType | ServerMetricType,
    ContextResourceData<PrometheusQueryRangeResultValue | PrometheusQueryRangeResponseDataResult>
  >;
  refresh: () => void;
  namespace: string;
};
```

We could do something like this (maybe there's a less cumbersome way or better names, but you get the idea):
```ts
export type MetricTypeWithQueryRangeResultValue =
  | ServerMetricType.REQUEST_COUNT
  | ServerMetricType.CPU_UTILIZATION
  | ServerMetricType.MEMORY_UTILIZATION
  | ModelMetricType.REQUEST_COUNT_SUCCESS
  | ModelMetricType.REQUEST_COUNT_FAILED;

export type MetricTypeWithQueryRangeResponseDataResult =
  | ServerMetricType.AVG_RESPONSE_TIME
  | ModelMetricType.TRUSTY_AI_SPD
  | ModelMetricType.TRUSTY_AI_DIR;

export type ModelServingMetricsData = Record<
  MetricTypeWithQueryRangeResultValue,
  ContextResourceData<PrometheusQueryRangeResultValue>
> &
  Record<
    MetricTypeWithQueryRangeResponseDataResult,
    ContextResourceData<PrometheusQueryRangeResponseDataResult>
  >;

type ModelServingMetricsContextType = {
  data: ModelServingMetricsData;
  refresh: () => void;
  namespace: string;
};
```

That way we can't screw ourselves up by making the wrong type assertion when consuming the data.